hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -191,11 +207,31 @@ private void registerTarget(final Message message, final String virtualHost) {
         final Target target = controllerManagement.findOrRegisterTargetIfItDoesNotexist(thingId, amqpUri);
         LOG.debug("Target {} reported online state.", thingId);
 
-        lookIfUpdateAvailable(target);
+        sendCurrentUpdateCommandToTarget(target);
     }
 
-    private void lookIfUpdateAvailable(final Target target) {
+    private void sendCurrentUpdateCommandToTarget(final Target target) {
```

Comment:
```suggestion
    private void sendUpdateCommandToTarget(final Target target) {
```
You decide within this method if it is "current" or "oldest"